### ISSUE TITLE
אין NPC בכלל בקלאס הול של האנטר

### ISSUE THREAD

**Issue by Sucoman** [LegionIsrael/LegionIsraelBugTracker#47]
אין בכלל אנשים בקלאס הול של האנטר, יש רק 2 אנשים שאחד של הסיל ואחד שעושה ריקרוט, חוץ מזה הקלאס הול ריק לחלוטין.

**Comment by LegionIsrael** [comment]
בתכלס הרבה מהקלאס הול לא באמת עוזר לשחקן, יש רק 4 NPCS חשובים בכל הקלאס הול

בכל זאת, אנחנו מחכים לסניפים על מנת למלא את החסר, מאמין שכמה ימים זה יסתדר כבר

**Comment by Sucoman** [comment]
כן, אבל תבינו שאני לא יכול לקנות את הגיר ההתחלתי שם, כי אין לי עם מי.
